packages/venia-concept/templates/open-document.mst

```diff
@@ -1,5 +1,5 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" data-image-optimizing-origin="{{env.IMAGE_OPTIMIZING_ORIGIN}}" data-backend="{{env.MAGENTO_BACKEND_URL}}">
```

Comment:
Would it be better to store these in a `meta` tag rather than directly on the root?